install/tools/ipa-replica-manage.in

```diff
@@ -358,9 +357,14 @@ def del_link(realm, replica1, replica2, dirman_passwd, force=False):
 
     return True
 
-def get_ruv(realm, host, dirman_passwd, nolookup=False, ca=False):
+
+def get_ruv(realm, host, dirman_passwd, nolookup=False, ca=False,
+            strict=True):
```

Comment:
Looks like strict is never True. Is it necessary? Future proofing?

Reply:
> Looks like strict is never True. Is it necessary? Future proofing?

It is in set to True in:  list_ruv() and get_rid_by_host()

I added it because I don't want to introduce any possible regressions when a decoding failure should not be ignored.  Is it needed, not sure, but I figured it was best to not change the behavior across the board.